micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jetty/JettyConnectionMetrics.java

```diff
@@ -0,0 +1,76 @@
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.binder.MeterBinder;
+import org.eclipse.jetty.io.ConnectionStatistics;
+
+public class JettyConnectionMetrics implements MeterBinder {
+
+    private final ConnectionStatistics connectionStatistics;
+    private final Iterable<Tag> tags;
+
+    public JettyConnectionMetrics(ConnectionStatistics connectionStatistics, Iterable<Tag> tags) {
+        this.connectionStatistics = connectionStatistics;
+        this.tags = tags;
+    }
+
+    @Override
+    public void bindTo(MeterRegistry registry) {
+        Gauge.builder("jetty.connector.connections.current", connectionStatistics, ConnectionStatistics::getConnections)
+                .tags(tags)
+                .description("The current number of open connections")
+                .register(registry);
+        Gauge.builder("jetty.connector.connections.max", connectionStatistics, ConnectionStatistics::getConnectionsMax)
+                .tags(tags)
+                .description("The maximum number of connections")
+                .register(registry);
+        Gauge.builder("jetty.connector.connections.total", connectionStatistics, ConnectionStatistics::getConnectionsTotal)
+                .tags(tags)
+                .description("The total number of connections")
+                .register(registry);
+
+        Gauge.builder("jetty.connector.bytesReceived", connectionStatistics, ConnectionStatistics::getReceivedBytesRate)
+                .tags(tags)
+                .description("The rate of bytes received")
+                .baseUnit("bytes/second")
+                .register(registry);
+        Gauge.builder("jetty.connector.bytesSent", connectionStatistics, ConnectionStatistics::getSentBytesRate)
+                .tags(tags)
+                .description("The rate of bytes sent")
+                .baseUnit("bytes/second")
+                .register(registry);
+
+        Gauge.builder("jetty.connector.duration.max", connectionStatistics, ConnectionStatistics::getConnectionDurationMax)
+                .tags(tags)
+                .description("The maximum connection lifetime duration")
+                .baseUnit("milliseconds")
+                .register(registry);
+        Gauge.builder("jetty.connector.duration.mean", connectionStatistics, ConnectionStatistics::getConnectionDurationMean)
+                .tags(tags)
+                .description("The mean connection lifetime duration")
+                .baseUnit("milliseconds")
+                .register(registry);
+        Gauge.builder("jetty.connector.duration.stddev", connectionStatistics, ConnectionStatistics::getConnectionDurationStdDev)
+                .tags(tags)
+                .description("The standard deviation of the connection lifetime duration")
+                .baseUnit("milliseconds")
+                .register(registry);
```

Comment:
Do we want to use https://github.com/micrometer-metrics/micrometer/blob/master/micrometer-core/src/main/java/io/micrometer/core/instrument/TimeGauge.java#L25 ?

Reply:
Yes, I think a TimeGauge will be better.